### DISCUSSION TITLE
rAudio 1 • 20220423: Bluetooth vanished from Networks menu

### DISCUSSION THREAD

**Discussion by klassikmann** [rern/rAudio#903] [Bug Reports]
The BT configuration is no more available in the network tab. I'm not sure if the menu was still there in the last release.

pacman gives a warning message:
warning: bluez-alsa: local (3.1.0.r106.g177e163-1) is newer than +R (3.1.0.r2.g07c6afc-1)

**Comment by ceejayemm** [comment]
I can confirm the problem reported by @klassikmann .

After updating to 20220423 and rebooting the RPi the option to enable BlueTooth on the System page is disabled and all attempts to turn it on fail hence no Bluetooth devices are available on the Networks page to connect to.  The Bluetooth connections were available under 202204151 on the Networks page.

Chris

**Comment by hjfisch** [comment]
Can confirm too.
Bluetooth is gone after rebbot.
My phone could not find the device to play from also.
When I enable Bluetooth in System it automaticly switches off.

**Comment by hjfisch** [comment]
Bluetooth update from today is not shown

**Comment by rern** [comment]
Please try today's update.

**Reply by klassikmann** [reply]
rAudio 1 • 20220428: BT menu back again. rAudio as receiver tested successfully.

EDIT: But disconnecting the playback menu remains in BT mode with default icon. Screenshot follows.

EDIT2:
<img src="https://user-images.githubusercontent.com/25383513/165772885-c5e8e4ea-f61f-4c4d-8120-853ea14cbb68.jpg" width=300>
